projects/eslint-config/plugins/liferay/lib/rules/no-abbreviations.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const ABBREVIATIONS_REPLACEMENTS = {
```

Comment:
I found [this repo](https://github.com/kisvegabor/abbreviations-in-code) here containing some common abbreviations that might look interesting. I found some missing abbreviations that I caught myself using like:
```
...
cfg: ['configuration']
cmd: ['command']
col: ['column']
coord: ['coordinate']
ctx: ['context']
func: ['function']
len: ['length']
msg: ['message']
param: ['parameter']
pos: ['position']
res: ['result']
src: ['source']
...
```

Does make sense including on the Map?

Reply:
I would say we should keep adding abbreviations as we find them in reviews. Ideally it should settle some day into a list containing team's biases :sweat_smile:

Reply:
Makes sense to me... although for now I just added the ones written in our [guidelines](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/naming.md#avoid-abbreviations). 
Some of them (`cfg, col, msg, res`...) could be very common to find in portal (I guess)... so what is your opinion, should we add them now or wait to find them in portal? Personally, I would add them now and update our guidelines...

Reply:
> Personally, I would add them now and update our guidelines...

I agree, though I don't know how much work that is. So maybe we don't want to add (now) those that appear in myriad places. Especially since fixing format issues here is easy, but that's not the case for `liferay-portal` :-(